tasks/2017/spread-the-word.md

```diff
@@ -0,0 +1,83 @@
+# GCI Tasks: Spread The Word
+
+## Background
+
+Upvotes and reviews make a big difference in the public perception of projects
+like Zulip.
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites.
+You're free to decide whether or not to do each individual task, but we do ask
+that you do at least five of the social media marketing tasks in order to get
+credit for the overall task. We expect completing all the tasks will take about
+90 minutes.
+
+Start by opening up a text file. You will be putting links and screenshots in
+this file in order to prove that you have completed parts of the task. We'll
+call this file the submission file.
+
+### Star us on GitHub.
+
+There are four main repositories:
+[server/web](https://github.com/zulip/zulip),
+[mobile](https://github.com/zulip/zulip-mobile),
+[desktop](https://github.com/zulip/zulip-electron), and [Python
+API](https://github.com/zulip/python-zulip-api). For each repository, star the
+repository, and then write down the number of stars that repository has in
+your submission file.
+
+### Share Zulip on StackShare
+
+1. Create an account at StackShare. Add your name and a picture, [i.e. do the
+work to have a real profile].
+
+2. Think of six technology products you really like, and consider adding Zulip
+to that list.
+
+3. For each product in 2: Add the product to your Stack, star it, and upvote at
+least three reasons why people like product that you find compelling.
+
+4. Add a link to your profile to your submission file. The link should look like
+https://stackshare.io/Lyla-Fischer, but replace Lyla-Fischer with your
+username.
+
+### Write a review on the android play store
```

Comment:
Android Play Store? I think it's Google Play Store and we should probably keep capitalization.